draft-ietf-quic-http.md

```diff
@@ -530,17 +536,28 @@ request using the corresponding stream ID, a server push using a Push ID (see
 The values for the Prioritized Element Type and Element Dependency Type imply
 the interpretation of the associated Element ID fields.
 
-| Type Bits | Type Description | Element ID Contents |
-| --------- | ---------------- | ------------------- |
-| 00        | Request stream   | Stream ID           |
-| 01        | Push stream      | Push ID             |
-| 10        | Placeholder      | Placeholder ID      |
-| 11        | Root of the tree | Ignored             |
+| Type Bits | Type Description | Prioritized Element ID Contents |
+| --------- | ---------------- | ------------------------------- |
+| 00        | Request stream   | Stream ID                       |
+| 01        | Push stream      | Push ID                         |
+| 10        | Placeholder      | Placeholder ID                  |
+| 11        | Current stream   | Absent                          |
+
+| Type Bits | Type Description | Element Dependency ID Contents |
+| --------- | ---------------- | ------------------------------ |
+| 00        | Request stream   | Stream ID                      |
+| 01        | Push stream      | Push ID                        |
+| 10        | Placeholder      | Placeholder ID                 |
+| 11        | Root of the tree | Absent                         |
 
 Note that the root of the tree cannot be referenced using a Stream ID of 0, as
 in {{!RFC7540}}; QUIC stream 0 carries a valid HTTP request.  The root of the
-tree cannot be reprioritized. A PRIORITY frame that prioritizes the root of the
-tree MUST be treated as a connection error of type HTTP_MALFORMED_FRAME.
+tree cannot be reprioritized. A PRIORITY frame sent on a request stream that
+prioritizes any other stream or expresses a dependency on a request with a
+greater Stream ID than the current stream MUST be treated as a stream error of
+type HTTP_MALFORMED_FRAME.  Likewise, a PRIORITY frame sent on a control stream
+that prioritizes the current stream MUST be treated as a connection error of
+type HTTP_MALFORMED_FRAME.
```

Comment:
I am not sure what the intent of the last sentence is.

My understanding is that a PRIORITY frame can only refer to a "Stream ID of a request stream" (assuming that the type bits are `00` (i.e. Request stream). Therefore, I think that we do not need this sentence.

Reply:
This is the inverse of the above -- sending `11` on a control stream, indicating that the thing being prioritized is the control stream itself.